### DISCUSSION TITLE
How to add uno extensions to a project that created by vs uno templates?

### DISCUSSION THREAD

**Discussion by garawaa** [unoplatform/uno#10805] [Q&A]
I tried to add some uno extensions through vs nuget package manager? But failed.

**Comment by jeromelaban** [comment]
@nickrandolph can certainly help!

**Comment by nickrandolph** [comment]
@garawaa the easiest way is to start with the unoapp-extensions template that is part of the dotnet new templates (https://www.nuget.org/packages/Uno.Extensions.Templates).  Alternatively the [Hosting docs](https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/Hosting/HostingOverview.html) provide a good starting point
